src/orchestratord/src/controller/materialize/environmentd.rs

```diff
@@ -1174,40 +1181,40 @@ fn create_environmentd_statefulset_object(
         args.push("--tls-mode=disable".to_string());
     }
     if let Some(ephemeral_volume_class) = &config.ephemeral_volume_class {
-        args.extend([
-            format!(
-                "--orchestrator-kubernetes-ephemeral-volume-class={}",
-                ephemeral_volume_class
-            ),
-            "--scratch-directory=/scratch".to_string(),
-        ]);
-        volumes.push(Volume {
-            name: "scratch".to_string(),
-            ephemeral: Some(EphemeralVolumeSource {
-                volume_claim_template: Some(PersistentVolumeClaimTemplate {
-                    spec: PersistentVolumeClaimSpec {
-                        access_modes: Some(vec!["ReadWriteOnce".to_string()]),
-                        storage_class_name: Some(ephemeral_volume_class.to_string()),
-                        resources: Some(VolumeResourceRequirements {
-                            requests: Some(BTreeMap::from([(
-                                "storage".to_string(),
-                                mz.environmentd_scratch_volume_storage_requirement(),
-                            )])),
+        args.push(format!(
+            "--orchestrator-kubernetes-ephemeral-volume-class={}",
+            ephemeral_volume_class
+        ));
+        if !mz.meets_minimum_version(&V153_DEV0) {
+            args.push("--scratch-directory=/scratch".to_string());
+            volumes.push(Volume {
+                name: "scratch".to_string(),
+                ephemeral: Some(EphemeralVolumeSource {
+                    volume_claim_template: Some(PersistentVolumeClaimTemplate {
+                        spec: PersistentVolumeClaimSpec {
+                            access_modes: Some(vec!["ReadWriteOnce".to_string()]),
+                            storage_class_name: Some(ephemeral_volume_class.to_string()),
+                            resources: Some(VolumeResourceRequirements {
+                                requests: Some(BTreeMap::from([(
+                                    "storage".to_string(),
+                                    mz.environmentd_scratch_volume_storage_requirement(),
+                                )])),
+                                ..Default::default()
+                            }),
```

Comment:
Since this was optional and unused do we even need to push the arg for prior versions?

Reply:
I've just tested with v0.130.13, which is the version our upgrade tests use, and it runs fine without the scratch volume. v0.130.0 doesn't work with the latest orchestratord for other reasons, so I'm not bothering to check it.